.github/workflows/unit-tests.yml

```diff
@@ -15,7 +15,8 @@ env:
   TEST_DATA_UPLOAD_PATH: input/docs_test_subset.json
   INGEST_OUTPUT_PREFIX: ingest_unit_test_parser_input
   DOCUMENT_NAME_KEY: document_id
-  TEST_DATA_EXPECTED_OUTPUT_FILE_PATH: test/data/docs_test_subset_parser_input_expected.json
+  PARSER_INPUT_EXPECTED_DATA_FILE_PATH: test/data/docs_test_subset_parser_input_expected.json
```

Comment:
What's the significance of "expected" in the variable names here? (i.e. what does it add?)

Reply:
This is the json object that we 'expect' to exist in the parser input after running the test and thus assert that the output meets.